less/middle-content.less

```diff
@@ -0,0 +1,320 @@
+.home {  
+    margin-left: 7%;
+    margin-right: 7%;
+
+    @media @phone {
+      margin-top: 10%;
+    }
+  
+    .middle-content {    
+      height: 100vh;
```

Comment:
I'm getting some really weird margin issues, and this is one of the problems. Height of 100vh will give this section the height of the viewport. So if I load your page on a laptop, the height will be quite a bit different than if I load it on a desktop. You can just get rid of this line and the container will automatically size to fit whatever is inside of it.